.gcp/release-docker.yaml

```diff
@@ -57,9 +57,13 @@ steps:
     args:
       - -c
       - |
-        export GEMINI_SANDBOX_IMAGE_TAG=$$(cat /workspace/image_tag.txt)
-        echo "Using Docker image tag for publish: $$GEMINI_SANDBOX_IMAGE_TAG"
-        npm run publish:sandbox
+        IMAGE_TAG=$(cat /workspace/image_tag.txt)
+        BASE_IMAGE_URI=$(npm run -s config get sandboxImageUri)
+        IMAGE_URI_NO_TAG=$(echo $BASE_IMAGE_URI | cut -d: -f1)
+        FINAL_IMAGE_URI="${IMAGE_URI_NO_TAG}:${IMAGE_TAG}"
+
+        echo "Pushing sandbox image: ${FINAL_IMAGE_URI}"
+        $_CONTAINER_TOOL push "${FINAL_IMAGE_URI}"
```

Comment:
this change is to reduce the npm scxripts; this publish step can or should be run locally, so moving it to just the release process.